contrib/msggen/msggen/gen/rust.py

```diff
@@ -271,6 +274,13 @@ def generate_request_trait_impl(self, method: Method):
             type Response = super::responses::{method.response.typename};
         }}
 
+        impl TypedRequest for {method.request.typename} {{
+            type Response = super::responses::{method.response.typename};
```

Comment:
I love the fact that you're using associated types to determine the response type from the request type :+1: Very nice ^^